### ISSUE TITLE
fmgr_pm_pkg_adom Can't Delete Package - State is considered an unsupported parameter

### ISSUE THREAD

**Issue by rrobles12** [fortinet-ansible-dev/ansible-galaxy-fortimanager-collection#18]
The issue is present with the following modules:
fmgr_pm_pkg_adom 
fmgr_pm_pkg
fmgr_pm_pkg_global

Trying to delete a package and the documentation [here ](https://ansible-galaxy-fortimanager-docs.readthedocs.io/en/galaxy-2.0.1/docgen/fmgr_pm_pkg.html )states:

To create or update an object, use state: present directive.
To delete an object, use state: absent directive

I try this even though it is not listed in the example and I get the following error message:

`"Unsupported parameters for (fmgr_pm_pkg) module: state Supported parameters include: adom, bypass_validation, pkg_path, pm_pkg, rc_failed, rc_succeeded, workspace_locking_adom, workspace_locking_timeout"}`

I've also tried under pkg_path but the same error occurs. Where am I suppose to put the state section? This is true for all the modules listed above.

Playbook:

```
  tasks:
  - name: DELETE POLICY PACKAGE
    fmgr_pm_pkg:
      adom: "{{ adom }}"
      pkg_path: "{{ pkg }}"
      state: absent
      pm_pkg:
        name: "{{ pkg }}"
```

```
ansible --version
ansible 2.10.5

ansible-galaxy collection list
Collection            Version
--------------------- -------
ansible.netcommon     1.2.1
fortinet.fortimanager 2.0.1

```

**Comment by chillancezen** [comment]
Good Issue.   will take look into it. 
ETA: Feb 5

**Comment by chillancezen** [comment]
the fix will be included in FMG galaxy collection `2.0.3`.
IT WILL BE RELEASED IN A FEW DAYS.

see document for more: https://ansible-galaxy-fortimanager-docs.readthedocs.io/en/galaxy-2.0.3/faq.html#modules-for-policy-package

**Comment by chillancezen** [comment]
`2.0.3` has been released.
install it by `ansible-galaxy collection install -fn fortinet.fortimanager`
